pkg/models/cluster.go

```diff
@@ -5,6 +5,14 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+const (
+	// from https://github.com/EnterpriseDB/dataplane-postgres-operator/blob/83d333f137280e8c1c7b27470f469aa58b0f763e/api/v1beta1/dataplanepostgrescluster_types.go#L301
```

Comment:
@nstott do you want to reference to a private repository here? Also, in the other comments, I'm not pointing all.